src/nvim/tui/input.h

```diff
@@ -15,8 +15,8 @@ typedef struct term_input {
   bool waiting;
   bool ttimeout;
   int8_t waiting_for_bg_response;
-  int8_t waiting_for_extkeys_response;
-  bool supports_kitty_protocol;
+  int8_t waiting_for_csi_u_response;
+  bool supports_csi_u;
```

Comment:
the space before the "u" seems kinda distracting.
```suggestion
  bool supports_csiu;
```